examples/flow/stokesNonNewtonian.ipynb

```diff
@@ -515,7 +515,20 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.10"
+   "version": "3.9.5"
+  },
+  "toc": {
```

Comment:
Do you have some sort of add on to your notebook that's adding this section? I don't see a table of contents. It's not important. Opening and running the notebook is just fine for me.

Reply:
https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/toc2